radicle-node/src/runtime/handle.rs

```diff
@@ -64,6 +65,24 @@ pub struct Handle<G: Signer + Ecdh> {
 
     /// Whether a shutdown was initiated or not. Prevents attempting to shutdown twice.
     shutdown: Arc<AtomicBool>,
+
+    /// Publishes events to subscribers.
+    emitter: Emitter<Event>,
+}
+
+trait EventEmitter {
+    fn subscribe(&mut self) -> chan::Receiver<Event>;
+}
```

Comment:
Can we avoid the trait?

Reply:
it's there to suppress:
```rust
warning: associated function `subscribe` is never used
   --> radicle-node/src/runtime/handle.rs:100:8
    |
100 |     fn subscribe(&mut self) -> chan::Receiver<Event> {
    |        ^^^^^^^^^
    |
    = note: `#[warn(dead_code)]` on by default
```

Reply:
We can just move it to `Emitter` and make it public no?

Reply:
https://github.com/radicle-dev/heartwood/pull/412/commits/1fe751e262acbe8c687420cd4ff8357099494e9e